docs/modules/ROOT/pages/references/architecture/upgrade_controller.adoc

```diff
@@ -221,6 +247,12 @@ spec:
           - openshift-monitoring
           customQueries:
           - query: "up{job=~"^argocd-.+$",namespace="syn"} != 1"
+        machineConfigPools: <8>
+        - matchLabels:
+            name: x-app-night-maintenance
+          delayUpgrade:
+            startAfter: "1h"
+            startBefore: "2h"
```

Comment:
```suggestion
            delayMin: "1h"
            delayMax: "2h"
```
`startAfter` and `startBefore` are dates in the UpgradeJob, I find it confusing if the same keys are used for relative times here

Reply:
`delayMin/Max` that was also my first idea and then went forth and back 😅